catroid/src/main/java/org/catrobat/catroid/content/XmlHeader.java

```diff
@@ -48,7 +49,7 @@
 	private float catrobatLanguageVersion;
 	private boolean landscapeMode;
 	@SuppressWarnings("unused")
-	public boolean scenesEnabled = true;
+	public String scenesEnabled = Constants.SCENES_ENABLED_TAG;
```

Comment:
Keep the boolean :wink: 

I did not test it, but I guess this member will be converted to
`<scenesEnabled>&lt;scenesEnabled&gt;</scenesEnabled>` or whatever the greater, less than signs are encoded to.
So, it's fine to use true/false.

However, previously you just used `scenesEnabled` to search for the tag which may be a string as well. When looking for a tag, use the tag symbols as well `<scenesEnabled>`.